nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -259,11 +277,24 @@ BtNavigator::navigateToPose()
       action_server_->publish_feedback(feedback_msg);
     };
 
+  auto bt_xml_filename = action_server_->get_current_goal()->behavior_tree;
+
+  // Empty id in request is default for backward compatibility
+  if (bt_xml_filename == "") {
+    bt_xml_filename = default_bt_xml_filename_;
+  }
+
+  if (!loadBehaviorTree(bt_xml_filename)) {
+    RCLCPP_WARN(
+      get_logger(), "BT id not found: %s. Using previous %s",
```

Comment:
I'm not sure the ID is the right word, wouldn't it be BT XML file or something?

Reply:
Yes, I forgot to change it when I used IDs instead of files. Changed